<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zgnep_msg</class>
 <widget class="QWidget" name="zgnep_msg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>400</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/src/title_bg.png);</string>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;color: rgb(255, 255, 255); background:transparent;</string>
    </property>
    <property name="text">
     <string>知更鸟教育平台-提示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>6</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px; background:transparent; background-image: url(:/src/close_normal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>400</width>
     <height>120</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>400</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0; background:transparent; background-image: url(:/src/tips_icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tips">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0; background:transparent; color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>这是一个测试信息！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0; background:transparent; background-image: url(:/src/confirm_normal.png);</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0; background:transparent; background-image: url(:/src/cancel_normal.png);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
